src/BigQuery/BigQueryClient.php

```diff
@@ -178,6 +178,21 @@ public function query($query, array $options = [])
         ))->query($query);
     }
 
+    /**
+     * As this method is an alias, please see
```

Comment:
I would include the method description from query() as well, although I think eliding the usage examples makes sense.

Reply:
updated